tutorbackup/templates/backup/build/backup/backup_services.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+
+import hashlib
+import logging
+import os
+import shutil
+import sys
+import tarfile
+from subprocess import check_call
+
+import click
+from botocore.exceptions import ClientError
+
+
+ENV = os.environ
+
+DUMP_DIRECTORY = '/data'
+
+MYSQL_DUMPFILE = os.path.join(DUMP_DIRECTORY, 'mysql_dump.sql')
+MONGODB_DUMPDIR = os.path.join(DUMP_DIRECTORY, 'mongodb_dump')
+CADDY_DUMPDIR = os.path.join(DUMP_DIRECTORY, 'caddy')
+
+TARFILE = '/backup/backup.tar.xz'
+
+logger = logging.getLogger(__name__)
+
+PERFORM_CADDY_BACKUP = os.path.isdir('caddy')
```

Comment:
I'm a bit confused why `flake8` doesn't flag this (I checked; it doesn't), but `PERFORM_CADDY_BACKUP` is now no longer used anywhere and can be removed.

Reply:
Good catch! Strange that it was not flagged